### ISSUE TITLE
Spring back into icon animation broken/stuck on A15 QPR1

### ISSUE THREAD

**Issue by D3SOX** [Deletescape-Media/Lawnchair#5094]
### Describe the bug

The animation when going back to the homescreen that it springs back into the last opened app's icon is broken. It shows a duplicate icon.

### Steps to reproduce

1. Open an app from the homescreen
2. Go back to the homescreen
3. See error

### Expected behavior

Work as before

### Screenshots

https://github.com/user-attachments/assets/27d8db81-cf2f-4218-8ec4-0431c65e51ae

### Device information

- Device: Google Pixel 8 Pro
- OS: Android 15 QPR1, CalyxOS 6.2.1
- App version: f1a292495c13c04f2e2eb3c574943c1915baaad7

### Additional context

It happened after updating to Android 15 QPR1

**Comment by MrSluffy** [comment]
Did you disable system animation?

**Comment by MrSluffy** [comment]
https://github.com/user-attachments/assets/cf6f44ba-aeb5-4679-895e-c585b7792fda

**Comment by D3SOX** [comment]
After downgrading to 0d9768c I noticed the problem still occurs. So my assumption that it was caused by a LangChait update

**Comment by MrSluffy** [comment]
You using Android 15 qpr1?

**Comment by MrSluffy** [comment]
If yes A15 qpr1 broke it I had to wait for my device to have qpr1 to check what's causing it

**Comment by D3SOX** [comment]
Yes, just checked the changes and I have them.
These are the version details

<img width="250" src="https://github.com/user-attachments/assets/62653efa-cdf6-4cb9-bc9a-c62e938e9566">

**Comment by guniens** [comment]
Continues to happen in latest nightly

**Comment by D3SOX** [comment]
@guniens There was no fix pushed for this so I don't see the point in your comment

**Comment by pedrom34** [comment]
Same as https://github.com/LawnchairLauncher/lawnchair/issues/5081

**Comment by MrSluffy** [comment]
https://issuetracker.google.com/issues/366867005

**Comment by guniens** [comment]
Hi @MrSluffy 

Is it possible to have the same animation when using back gesture? Also both animations in app drawer too?
That would make it pretty much stock-like experience and a really good option for non-rooted users.

Thanks

**Comment by MrSluffy** [comment]
I will try to implement it (no promises)

**Comment by guniens** [comment]
Thanks a lot!

**Comment by D3SOX** [comment]
Sadly, the fix makes it worse for me. It now flickers upon leaving an app

https://github.com/user-attachments/assets/6d0508b5-eba9-436b-b991-0bf16d2ec979

**Comment by MrSluffy** [comment]
yeah it worse, you can turn it off in Settings -> Home -> Overlay

**Comment by D3SOX** [comment]
When I set it to None the animation works and looks good. The icon jumping up and down could be a bit shorter I think
